### DISCUSSION TITLE
Translate into Japanese.

### DISCUSSION THREAD

**Discussion by feliz-me** [SanjaySunil/BetterDiscordPanel#134] [Q&A] [answered]
I am Japanese, and I use BDP on a daily basis.
Despite the fact that BDP has a certain number of Japanese users, there is no translation into Japanese.
So, I am trying to translate it into Japanese.

I have read translate.md and understand how to commit.
Is there any problem in doing this translation?
If no, I'll commit and pull request.

**Comment by SanjaySunil** [comment, marked as answer]
I'm currently not using any translations for the control panel (BetterDiscordPanel.cmd) as I need to make some changes to it when I have time.

However, it would be greatly appreciated if translations can be made for the actual [BDP web panel](https://github.com/SanjaySunil/BetterDiscordPanel/blob/master/.github/TRANSLATION.md#1-open-the-translation-file-localesjs).
